### PR TITLE
Blog Application

### PR DESCRIPTION
**Pull request by khushi-purwar**
#137  Added the blog application. Please review my PR @Ayushparikh-code 

ss1

### CODE REVIEW

File: Blog Application/style.css [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+    height: 300px;
+    width: auto;
+}
+
+.place__holder__image{
+    width: 100%;
+}
+
+
+
```
Comment (chicken-biryani) [inline Blog Application/style.css:23]: Use a css beautifier.

File: Blog Application/script.js [hunk 2]
```diff
@@ -0,0 +1,250 @@
...
+
+	const getBlog = globalStore.filter(({
+		id
+	}) => id === targetID);
+	// console.log(getBlog[0]);
+	blogModal.innerHTML = htmlModalContent(getBlog[0]);
+};
+
+
+
```
Comment (chicken-biryani) [inline Blog Application/script.js:250]: Remove extra lines. Use a JS beautifier.

File: Blog Application/index.html [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+		</section>
+	</div>
+	<script src="https://cdn.jsdelivr.net/npm/@popperjs/core@2.9.2/dist/umd/popper.min.js" integrity="sha384-IQsoLXl5PILFhosVNubq5LC7Qb9DXgDA9i+tQ8Zj3iwWAwPtgFTxbJ8NT4GN1R8p" crossorigin="anonymous"></script>
+	<script src="https://cdn.jsdelivr.net/npm/bootstrap@5.0.1/dist/js/bootstrap.min.js" integrity="sha384-Atwg2Pkwv9vp0ygtn1JAojH0nYbwNJLPhwyoVbhoPwBhjQPR5VtM2+xf0Uwh9KtT" crossorigin="anonymous"></script>
+	<script src="script.js"></script>
+</body>
+
+</html>
+
+
```
Comment (chicken-biryani) [inline Blog Application/index.html:108]: Use an HTML beautifier.

File: Blog Application/README.md [hunk 4]
```diff
@@ -0,0 +1,30 @@
+# Blog Application
+
+## Description
+Blog Application is an application where user can add a new blog, edit it, delete it as well as view other blogs and make changes in them. 
+
+-> Application uses LocalStorage, the localStorage object stores data with no expiration date. The data will not be deleted when the browser is closed, and will be available when the user comes again.
+
```
Comment (chicken-biryani) [inline Blog Application/README.md:7]: Add a "Use of projectt" section.

File: Blog Application/README.md [hunk 5]
```diff
@@ -0,0 +1,30 @@
+# Blog Application
+
+## Description
+Blog Application is an application where user can add a new blog, edit it, delete it as well as view other blogs and make changes in them. 
+
+-> Application uses LocalStorage, the localStorage object stores data with no expiration date. The data will not be deleted when the browser is closed, and will be available when the user comes again.
```
Comment (chicken-biryani) [inline Blog Application/README.md:6]: Remove the ->

**Review by Ayushparikh-code — APPROVED** [review]

File: Blog Application/index.html [hunk 6]
```diff
@@ -0,0 +1,112 @@
...
+					</li>
+				</ul>
+				<button type="button" class="btn btn-primary rounded-pill" data-bs-toggle="modal" data-bs-target="#staticBackdrop"><i class="fas fa-plus"></i> Add New</button>
+			</div>
+		</div>
+	</nav>
+	<div class="container">
+		<section>
+			<div class="row blog__container mt-5 mb-3 ">
+				<!-- <div class="col-lg-4 col-md-6">
```
Comment (chicken-biryani) [inline Blog Application/index.html:85]: Is the comment required? If not then I think you can remove it. The rest looks really good. Good work @khushi-purwar !

Comment (khushi-purwar) [reply Blog Application/index.html:85]: No, it is not required. I will remove it.

**Pull request merged**